tests/migrations/test_operations.py

```diff
@@ -2437,3 +2438,60 @@ class TestCreateModel(SimpleTestCase):
 
     def test_references_model_mixin(self):
         CreateModel('name', [], bases=(Mixin, models.Model)).references_model('other_model')
+
+
+class FieldOperationTests(SimpleTestCase):
+    def test_references_model(self):
+        operation = FieldOperation('MoDel', 'field')
+        # When missing a field declaration always assume it's referencing.
+        self.assertTrue(operation.references_model('Whatever'))
```

Comment:
Do you prefer `self.assertTrue` to `self.assertIs(..., True)` for these tests. Sometimes the former can give incorrect results if a non-bool is returned.